test/common.js

```diff
@@ -234,6 +232,41 @@ exports.saveCore = function saveCore(options, cb) {
   });
 }
 
+function saveCoreLinux(executable, scenario, core, cb) {
+  const cmd = `ulimit -c unlimited && ${executable} ` +
+              `--abort_on_uncaught_exception --expose_externalize_string ` +
+              `${path.join(exports.fixturesDir, scenario)}; ` +
+              `mv ./core ${core}`;
```

Comment:
Can you split this into a separate command? It is possible that the core is not `./core` depending on the configuration of the system

Reply:
Splitted into a separate command, but I think we should handle different configurations on a separate PR.